### DISCUSSION TITLE
Serious Problems Found Here! — Atom

### DISCUSSION THREAD

**Discussion by alircxa** [atom/atom#22910] [General]
Hello, my atomic friends!

There is a bunch of serious problem in Atom website that I want to share with you!

1. I want to report these on chat, but, On the index page when I clicked on the "[Chat — Slack](http://atom-slack.herokuapp.com/)" link, it goes to somewhere that is not available. Look at this screenshot: **[Link](https://anonfiles.com/91M0A2E9ua/2021-08-29_20_41_13-Window_png)**

![2021-08-29 20_41_13-Window](https://user-images.githubusercontent.com/8685650/131361710-a13b8776-4dae-4b62-9d11-ae444267d669.png)

2. On all pages, (Packages, Themes, etc) all pictures that the users uploaded, are broken. At first, I assumed that it might be because of my adblocker or something (so I turned it off), but the problem was remains. I checked it out with GTMetrix and the answer was the same.

- **[My Screenshot](https://anonfiles.com/z0PaA3E6u0/2021-08-29_20_46_45-Window_png)** from **[This Page](https://atom.io/packages/tree-view-sort)** for example.

![2021-08-29 20_46_45-Window](https://user-images.githubusercontent.com/8685650/131361742-b4cd0060-2e42-48d5-9ad1-39c46ee1c667.png)

- **[GTMetrix](https://gtmetrix.com/reports/atom.io/4bTKDM7D/)** / **[Screenshot](https://gtmetrix.com/reports/atom.io/4bTKDM7D/screenshot.jpg)**

3. When I clicked on "[Forum — Discuss](https://discuss.atom.io/)" I faced **[this](https://anonfiles.com/F7R2A8Eeu9/2021-08-29_15_10_26-Window_png)** page. It seems there is some SSL inconsistency with GitHub.

![2021-08-29 15_10_26-Window](https://user-images.githubusercontent.com/8685650/131361769-c3569462-4e5c-49c0-be1f-f4ff3d889c4f.png)

Regards.

**Comment by icecream17** [comment]
Here's some more information

In the second one, the images come from "https://i.github-camo.com/".
I found https://stackoverflow.com/questions/57857193/github-image-without-camo
Nowadays when I paste an image the link comes from: https://user-images.githubusercontent.com/


The 3rd one is known

**Reply by alircxa** [reply]
Thanks for your reply. @icecream17
I hope they fix these issues. ASAP.
